Split/Network/Streaming/DefaultSseNotificationParser.swift

```diff
@@ -18,7 +18,7 @@ protocol SseNotificationParser {
 
     func parseMySegmentUpdate(jsonString: String) throws -> MySegmentsUpdateNotification
 
-    func parseOccupancy(jsonString: String) throws -> OccupancyNotification
+    func parseOccupancy(jsonString: String, timestamp: Int) throws -> OccupancyNotification
 
     func parseControl(jsonString: String) throws -> ControlNotification
```

Comment:
consider passing the timestamp to `parseControl`, like `parseOccupancy`

Reply:
Not sure. I think is better this way because we can avoid some steps while processing.